pom.xml

```diff
@@ -36,7 +36,7 @@
         <awaitilityVersion>4.0.3</awaitilityVersion>
         <hamcrest.version>1.3</hamcrest.version>
         <tsaasClientVersion>1.0.0</tsaasClientVersion><!-- tsaas via cloud gateway -->
-        <telemetryGatewayVersion>1.0.0-feature-DC-391-6</telemetryGatewayVersion>
+        <telemetryGatewayVersion>1.0.0-feature-DC-391-23</telemetryGatewayVersion>
```

Comment:
can update this to 1.2.2 if you'd prefer.

https://cloudsmith.io/~opennms/repos/maven/packages/detail/maven/telemetry-gateway-grpc/1.2.2/a=noarch;xg=org.opennms.tsaas.telemetry/

Reply:
:+1: that would look better

Reply:
no problem